client/src/app/site/pages/meetings/modules/list-of-speakers-content/components/list-of-speakers-content/list-of-speakers-content.component.ts

```diff
@@ -53,13 +52,8 @@ export class ListOfSpeakersContentComponent extends BaseMeetingComponent impleme
     public waitingSpeakers: ViewSpeaker[] = [];
     public activeSpeaker: ViewSpeaker | null = null;
 
-    /**
-     * Required for the user search selector
-     */
-    public addSpeakerForm: FormGroup;
-
-    public usersSubject = new BehaviorSubject<ViewUser[]>([]);
-    public filteredUsersSubject = new BehaviorSubject<ViewUser[]>([]);
+    public users: ViewUser[] = [];
+    public readonly nonAvailableUsersSubject = new BehaviorSubject<number[]>([]);
```

Comment:
Try something like this:

```suggestion
    public readonly nonAvailableUsersObservable = this.userRepository.getViewModelList().pipe(map(users
            .filter(
                user =>
                    !(!this.onlyPresentUsers || user.isPresentInMeeting()) ||
                    this.waitingSpeakers.some(speaker => speaker.user_id === user.id)
            )
            .map(user => user.id)))
```